examples/C++/DDS/DynamicHelloWorldExample/HelloWorldPublisher.cpp

```diff
@@ -113,15 +113,15 @@ void HelloWorldPublisher::PubListener::on_publication_matched(
         eprosima::fastdds::dds::DataWriter*,
         const eprosima::fastdds::dds::PublicationMatchedStatus &info)
 {
-    if(info.current_count_change > 0)
+    if(info.current_count_change == 1)
```

Comment:
This change (similar to the following ones) isn't really important, don't you think so.

Reply:
I think that due to how this structure is implemented, this change can help to identify future errors since the current_count_change value couldn't be greater than 1 or lower than -1

Reply:
It couldn't be 0 either, right?
In that case, maybe it will be a good idea to add an else branch showing an error and the received value.